server.js

```diff
@@ -19,4 +20,6 @@ app.post('/api/foods', FoodsController.create);
 app.patch('/api/foods/:id', FoodsController.update);
 app.delete('/api/foods/:id', FoodsController.destroy);
 
+app.get('/api/meals/:category/:year/:month/:day', MealsController.index);
```

Comment:
hey @TheKaterTot @cmacaulay - sorry for the delay - i think both ways are fine. if we want to be more RESTful - i'd suggest the way that's passing the date via string params after the route (`/api/meals/:meal_name`). having four parameters in the url to keep track of is a lot